#### Parameters simulation ####

sheath_sim_pos: [0.0015, -0.039, 0.0]
sheath_sim_rot: [90.0, 0.0, 0.0]
length_init: 0.200
model_mesh_path: '/mesh/flat_model_empty.stl'
lcp_solv_mu: 1.e-2